cocos/ui/UIButton.cpp

```diff
@@ -369,7 +377,9 @@ void Button::loadTextureDisabled(const std::string& disabled,TextureResType texT
             _buttonDisabledRenderer->initWithFile(disabled);
             break;
         case TextureResType::PLIST:
-            _buttonDisabledRenderer->initWithSpriteFrameName(disabled);
+            // _buttonDisabledRenderer->initWithSpriteFrameName(disabled);
+            frm = SpriteFrameCache::getInstance()->getSpriteFrameByName(disabled);
+            _buttonDisabledRenderer->setSpriteFrame(frm);
```

Comment:
Confirm the issue existed, we need fix it. do you know why it happened? I am not sure this way fixing the bug is best.

one reason is in `initWithSpriteFrameName ` called `SpriteFrameCache::getInstance()->getSpriteFrameByName(disabled);`, `setSpriteFrame ` and the other method, so I think the bug is caused due to internal of `initWithSpriteFrameName`.

another reason is `initWithSpriteFrameName` do many checks, calling `getSpriteFrameByName ` and `setSpriteFrame ` directly do no checks, if I code:
```C++
spr->loadTextureNormal("Not Existed Texture", ui::Widget::TextureResType::PLIST);
```
the app will crash.

Reply:
I know the issue is within `initWithSpriteFrameName()` but I do not have time right now to debug. This is a temporary fix to make my project work.

Reply:
Understand, and I am doing something about cmake firstly, so let it stay here for now.